lib/timescaledb/migrations.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Membrane.RTC.Engine.TimescaleDB.Migrations do
+  @moduledoc """
+  Module providing functionality of performing migrations creating or deleting DB tables, expected by this library to function.
```

Comment:
```suggestion
  Migrations creating DB tables required by this library to function.
```
Some usage example is missing - both here and in README